velox/connectors/hive/HiveDataSink.h

```diff
@@ -192,11 +193,13 @@ class HiveInsertTableHandle : public ConnectorInsertTableHandle {
       std::shared_ptr<const LocationHandle> locationHandle,
       const dwio::common::FileFormat tableStorageFormat =
           dwio::common::FileFormat::DWRF,
-      std::shared_ptr<HiveBucketProperty> bucketProperty = nullptr)
+      std::shared_ptr<HiveBucketProperty> bucketProperty = nullptr,
+      std::optional<common::CompressionKind> compressionKind = {})
```

Comment:
Why you choose to use optional instead of rely on the actual compression kind (MAX) to detect if it has been set or not? Thanks!

Reply:
I realized I was wrong, the compressionKind is **not** in HiveOutputTableHandle already
It's introduced by us, so it's more straightforward to set it as optional